docs/source/using-executorch-ios.md

```diff
@@ -1,3 +1,324 @@
 # Using ExecuTorch on iOS
 
-Placeholder for top-level iOS documentation
\ No newline at end of file
+ExecuTorch supports both iOS and macOS via Objective-C, Swift, and C++. ExecuTorch also provides backends to leverage Core ML and Metal Performance Shaders (MPS) for hardware-accelerated execution on Apple platforms.
```

Comment:
Is this basically a move? 

Then do `git mv` to preserve the git history. And it's easier to review the small deltas (if any).

Reply:
Definitely. I did actually git mv in here. Looking at the blame on the branch, it looks like the history is preserved. I think it doesn't show up as a direct move in the PR since there are a few changes in the subsequent commit and the PR shows the aggregate deltas.